docs/notebooks/Berger_POPC_25ns_buildH_calcOP.ipynb

```diff
@@ -0,0 +1,1394 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "--2021-01-11 15:29:48--  https://raw.githubusercontent.com/patrickfuchs/buildH/first_notebook/def_files/order_parameter_definitions_MODEL_Berger_POPC.def\n",
+      "Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 151.101.120.133\n",
+      "Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|151.101.120.133|:443... connected.\n",
+      "HTTP request sent, awaiting response... 200 OK\n",
+      "Length: 2135 (2.1K) [text/plain]\n",
+      "Saving to: ‘order_parameter_definitions_MODEL_Berger_POPC.def’\n",
+      "\n",
+      "order_parameter_def 100%[===================>]   2.08K  --.-KB/s    in 0s      \n",
+      "\n",
+      "2021-01-11 15:29:49 (6.08 MB/s) - ‘order_parameter_definitions_MODEL_Berger_POPC.def’ saved [2135/2135]\n",
+      "\n",
+      "Berger_POPC_25ns_buildH_calcOP.ipynb\n",
+      "endCONF.gro\n",
+      "order_parameter_definitions_MODEL_Berger_POPC.def\n",
+      "popc0-25ns.trr\n"
+     ]
+    }
+   ],
+   "source": [
+    "!wget https://raw.githubusercontent.com/patrickfuchs/buildH/first_notebook/def_files/order_parameter_definitions_MODEL_Berger_POPC.def\n",
+    "!ls"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's have a look to this def file:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "gamma1_1 POPC C1  H11\n",
+      "gamma1_2 POPC C1  H12\n",
+      "gamma1_3 POPC C1  H13\n",
+      "gamma2_1 POPC C2  H21\n",
+      "gamma2_2 POPC C2  H22\n",
+      "gamma2_3 POPC C2  H23\n",
+      "gamma3_1 POPC C3  H31\n",
+      "gamma3_2 POPC C3  H32\n",
+      "gamma3_3 POPC C3  H33\n",
+      "beta1 POPC C5  H51\n"
+     ]
+    }
+   ],
+   "source": [
+    "!head order_parameter_definitions_MODEL_Berger_POPC.def"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The first column is the generic C-H name, second is the residue (lipid) name, third column the carbon PDB name, last column the hydrogen PDB name. This file will tell buildH which Hs you want to rebuild, and then to calculate the order parameter on the corresponding C-Hs. It means that it is possible to provide only a subset of possible C-Hs if, for example, you only want those from the polar heads. Note that if you want to output a trajectory (not done in this notebook), the def file must contain **all** possible C-Hs.\n",
```

Comment:
Je ne comprends pas la phrase 'Note that if you want to output a trajectory, the def file must contain all possible C-Hs'

Reply:
I tried to rephrase it 😄